AnyServe/AnyServe.ITests/UploadTests/ConstantString.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace AnyServe.ITests.UploadTests
+{
+    class ConstantString
```

Comment:
convert this class to be static
and set strings members as constants

We don't want to create instance of this class each time, instead the access to members will be like:
ConstansString.GerUrl